ansible_base/authentication/backend.py

```diff
@@ -28,7 +28,11 @@ def get_authentication_backends(last_updated):
 class AnsibleBaseAuth(ModelBackend):
     def authenticate(self, request, *args, **kwargs):
         logger.debug("Starting AnsibleBaseAuth authentication")
-        last_modified = Authenticator.objects.values("modified_on").order_by("-modified_on").first()["modified_on"]
+
+        # Query the database for the most recently last modified timestamp.
+        # This will be used as a cache key for the cached function get_authentication_backends below
+        last_modified_item = Authenticator.objects.values("modified_on").order_by("-modified_on").first()
+        last_modified = None if last_modified_item is None else last_modified_item.get('modified_on', None)
```

Comment:
```suggestion
        last_modified = None if last_modified_item is None else last_modified_item.get('modified_on')
```

You don't need the `None` here, it's the default.

Reply:
Fair enough, fix pushed.